### ISSUE TITLE
Makefile: llvm-config: Command not found

### ISSUE THREAD

**Issue by ghost** [Infinidat/infinitrace#4]
```
make: llvm-config: Command not found
```

Ubuntu names `llvm-config` as `llvm-config-3.3`. If I install the latest llvm, it will compile with errors.

``` bash
constantine@linux /mnt/space/dev/infinitrace $ llvm-config
The program 'llvm-config' is currently not installed. You can install it by typing:
sudo apt-get install llvm
constantine@linux /mnt/space/dev/infinitrace $ llvm-config-3.3
usage: llvm-config <OPTION>... [<COMPONENT>...]

Get various configuration information needed to compile programs which use
LLVM.  Typically called from 'configure' scripts.  Examples:
  llvm-config --cxxflags
  llvm-config --ldflags
  llvm-config --libs engine bcreader scalaropts

Options:
  --version         Print LLVM version.
  --prefix          Print the installation prefix.
  --src-root        Print the source root LLVM was built from.
  --obj-root        Print the object root used to build LLVM.
  --bindir          Directory containing LLVM executables.
  --includedir      Directory containing LLVM headers.
  --libdir          Directory containing LLVM libraries.
  --cppflags        C preprocessor flags for files that include LLVM headers.
  --cflags          C compiler flags for files that include LLVM headers.
  --cxxflags        C++ compiler flags for files that include LLVM headers.
  --ldflags         Print Linker flags.
  --libs            Libraries needed to link against LLVM components.
  --libnames        Bare library names for in-tree builds.
  --libfiles        Fully qualified library filenames for makefile depends.
  --components      List of all possible components.
  --targets-built   List of all targets currently built.
  --host-target     Target triple used to configure LLVM.
  --build-mode      Print build mode of LLVM tree (e.g. Debug or Release).
Typical components:
  all               All LLVM libraries (default).
  engine            Either a native JIT or a bitcode interpreter.
```

**Comment by ghost** [comment]
I'll probably alias it for now, but issue won't go away for others... :) Should be mentioned in readme, at least... Or update infinitrace to LLVM 3.4... :)

**Comment by baruch** [comment]
This was fixed in commit 31711a31a5273af940a4995cb591e445c3efd72e
